mkdocs/docs/HPC/getting_started.md

```diff
@@ -0,0 +1,209 @@
+See the [example](https://github.com/EESSI/eessi-demo/tree/main/TensorFlow).
+
+### Getting Connected
+
+There are two options to connect
+
+- Using a terminal(ssh) (for power users) (see [First Time connection to the HPC infrastructure](connecting.md#first-time-connection-to-the-hpc-infrastructure))
+- [Using the HPC-Ugent web portal](web_portal.md#using-the-hpc-ugent-web-portal)
+
+{%- if OS == windows %}
+
+Considering your operating system is **{{OS}}**, it's recommented to utilize the UGent web portal. This platform offers a convenient way to upload files and gain shell access.
+
+Ensure you can get to a shell before proceeding with the next steps.
+
+See [shell access](web_portal.md#shell-access) when using the ugent portal or
+[connection to the hpc infrastructur](connecting.md#first-time-connection-to-the-hpc-infrastructure) when using a terminal.
+
+!!! Info
+
+    When having problems see the [connection issues](troubleshooting.md#sec:connecting-issues) section on the [troubleshooting page](troubleshooting.md).
+
+{%- else %}
+
+Considering your operating system is **{{OS}}**, it's recommented to make use of a terminal with ssh to get the most flexibility. 
+
+Assuming you've already generated  SSH keys in the previous step ([Getting Access](#getting-access)), you should now be able to login by running the following command:
+
+<pre><code>ssh {{userid}}@{{loginnode}}</code></pre>
+
+!!! Warning
+    
+    replace <b>{{userid}}</b> with your id (see <https://account.vscentrum.be>)
+
+!!! Info
+
+    When having problems see the [connection issues](troubleshooting.md#sec:connecting-issues) section on the [troubleshooting page](troubleshooting.md).
+
+!!! Info
+
+    You can also still use the ugent web portal (see [shell access on web portal](web_portal.md#shell-access))
+    
+
+{%- endif %}
+
+### Transfer your files
+
+Now that you can login, it's time to transfer files from your local pc to your **Home Directory** on a cluster.
+
+Get [TensorFlow-2.x_mnist-test.py](https://raw.githubusercontent.com/EESSI/eessi-demo/main/TensorFlow/TensorFlow-2.x_mnist-test.py) 
+and [run.sh](https://raw.githubusercontent.com/EESSI/eessi-demo/main/TensorFlow/run.sh) from the [example](https://github.com/EESSI/eessi-demo/tree/main/TensorFlow) on your local machine.
+
+{%- if OS == windows %}
+
+The [HPC-Ugent web portal](https://login.hpc.ugent.be) provides a filebrowser that allows uploading files.
+For more information see the [file browser section](web_portal.md#file-browser).
+
+Upload the two files to your **Home Directory** and go back to your shell.
+
+!!! Info
+
+    As an alternative, you can use WinSCP (see [our section](connecting.md#winscp))
+
+{%- else %}
+
+On your local machine you can run:
+<pre><code>wget https://raw.githubusercontent.com/EESSI/eessi-demo/main/TensorFlow/TensorFlow-2.x_mnist-test.py
+wget https://raw.githubusercontent.com/EESSI/eessi-demo/main/TensorFlow/run.sh
```

Comment:
I think it's safer to assume that `curl` is there than `wget`, so use `curl -OL <url>` ?